packages/realm-app-importer/src/AdminApiClient.ts

```diff
@@ -379,14 +380,11 @@ export class AdminApiClient {
       if (typeof body === "object") {
         headers["content-type"] = "application/json";
       }
-      const response = await DefaultNetworkTransport.fetch(url, {
+      const response = await fetch(url, {
         ...rest,
-        body: JSON.stringify(body),
+        body: body ? JSON.stringify(body) : undefined,
```

Comment:
Nit: Accounting for if we want empty string, `0`, and `null` responses to not be interpreted as `undefined`.
```suggestion
        body: body === undefined ? undefined : JSON.stringify(body),
```

Reply:
The main use-case for this is actually to avoid calling stringify for empty strings (apparently cores way of signalling a missing body) and `undefined`.

Reply:
Thanks for the clarification 👍 I like your suggestion 🙂